crmd/throttle.c

```diff
@@ -102,14 +83,16 @@ int throttle_num_cores(void)
  *       This will return NULL if the daemon is being run via valgrind.
```

Comment:
cib and crmd both run as hacluster. We definitely depend on /proc being mounted with permissive options though. There's a similar problem for OSes without /proc as well. It's rare enough that it's up to users with that use case to submit a feature request and/or patch.